### ISSUE TITLE
quickfix: suggest grouping of global vars/consts

### ISSUE THREAD

**Issue by mvdan** [dominikh/go-tools#9]
```
var a int
var foo string = "bar"
var another = 123
```

should be

```
var (
    a          int
    foo string = "bar"
    another    = 123
)
```

On top of simpler lines, this introduces alignment. I'd say three contiguous var/const lines should trigger this. Empty lines or comments in between them should be ignored for the count.

**Comment by shurcooL** [comment]
I don't think we can make this suggestion. This is still too much of a human decision of what style to use, they are both roughly equivalent and have tradeoffs. The latter form makes sense if the variables are very related and you benefit from alignment. The former makes sense if the variables are unrelated and alignment isn't needed, and you prefer to save 2 vertical lines.

**Comment by shurcooL** [comment]
By the way, you have incorrect formatting in the grouped case. It should be:

```Go
var (
	a       int
	foo     string = "bar"
	another        = 123
)
```

**Comment by dominikh** [comment]
I'll have to agree with @shurcooL here. That aside, even if we agreed that variable grouping was the stylistically better choice in all cases, it'd be a question of style, not simplicity. The suggested alternative form isn't really any more simple. Thus, this would be something that golint could suggest, not gosimple.

**Comment by dominikh** [comment]
Reopening under the quickfix label.
